README.md

```diff
@@ -62,6 +62,7 @@ This constructor is used to transform an [API request](http://kuzzle.io/api-refe
 | `connectionId` | `string` | Passed to [RequestContext](#modelsrequestcontext) constructor |
 | `error` | `KuzzleError` or `Error` | Invokes [setError](#seterrorerror) at initialization |
 | `protocol` | `string` | Passed to [RequestContext](#modelsrequestcontext) constructor |
+| `headers` | `object` | Passed to [RequestContext](#modelsrequestcontext) constructor |
```

Comment:
Fifth option: change nothing to the current code, and simply add this line _after_ the `Request` object has been created:

`request.input.setHeaders(<headers>);`